<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Booru Tagger</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../assets/icon.png</normaloff>../assets/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="addToAllButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Add Tag to All</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="removeFromAllButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Remove Tag from All</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="allBelowCheckBox">
            <property name="maximumSize">
             <size>
              <width>55</width>
              <height>17</height>
             </size>
            </property>
            <property name="text">
             <string>Below</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="allAboveCheckBox">
            <property name="maximumSize">
             <size>
              <width>55</width>
              <height>17</height>
             </size>
            </property>
            <property name="text">
             <string>Above</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLineEdit" name="tagToRemoveLineEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QListWidget" name="images_list">
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="TableWidgetCompleter" name="image_tag_table">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="midLineWidth">
         <number>0</number>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="showGrid">
         <bool>true</bool>
        </property>
        <property name="gridStyle">
         <enum>Qt::SolidLine</enum>
        </property>
        <property name="rowCount">
         <number>1</number>
        </property>
        <property name="columnCount">
         <number>1</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>39</number>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>100</number>
        </attribute>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>38</number>
        </attribute>
        <row/>
        <column/>
        <item row="0" column="0">
         <property name="text">
          <string/>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="TableWidgetCompleter" name="all_images_tags_list">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="rowCount">
         <number>1</number>
        </property>
        <property name="columnCount">
         <number>1</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <row/>
        <column/>
        <item row="0" column="0">
         <property name="text">
          <string/>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>803</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="open_folder_action"/>
   </widget>
   <widget class="QMenu" name="menuPreview_Window">
    <property name="title">
     <string>Preview Window</string>
    </property>
    <addaction name="show_preview_action"/>
    <addaction name="show_tag_count_action"/>
   </widget>
   <widget class="QMenu" name="menuTemplates">
    <property name="title">
     <string>Templates</string>
    </property>
    <addaction name="open_template_action"/>
    <addaction name="separator"/>
    <addaction name="save_template_action"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="hotkeys_action"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPreview_Window"/>
   <addaction name="menuTemplates"/>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="open_folder_action">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="show_preview_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Preview Window</string>
   </property>
  </action>
  <action name="show_tag_count_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Tag Count</string>
   </property>
  </action>
  <action name="open_template_action">
   <property name="text">
    <string>Open Template</string>
   </property>
  </action>
  <action name="save_template_action">
   <property name="text">
    <string>Save Template</string>
   </property>
  </action>
  <action name="hotkeys_action">
   <property name="text">
    <string>Hotkeys</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TableWidgetCompleter</class>
   <extends>QTableWidget</extends>
   <header>Widgets.custom_table.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
